GitUI/UserControls/BlameControl.cs

```diff
@@ -66,6 +73,11 @@ public BlameControl()
             CommitInfo.CommandClicked += commitInfo_CommandClicked;
         }
 
+        public void SetRepositoryHostPlugin(IRepositoryHostPlugin gitHoster)
+        {
+            _gitHoster = gitHoster;
```

Comment:
If I interpret this correctly: This will not handle if a repo has remotes in multiple hosts, like GitHub and GitLab
Maybe not needed now, but add a comment that "last host is used".

Reply:
What I did is open to support other hosting services but at the moment we only have 1 repository host plugin.

So before having support for use of 2 host plugins at the same time, we will have to have 2 plugins 🤣 and we will have to change other things before: https://github.com/gitextensions/gitextensions/blob/master/GitUI/Plugin/PluginRegistry.cs#L56 (The current strategy is to use the first supported, and not the last)

Reply:
We will need to implement some kind of selector. But, like @pmiossec said, we'll cross the bridge once we have more than one host implementation.